packages/apps/manufacturing/src/types/manufacturing-order.tsx

```diff
@@ -152,35 +157,41 @@ class ManufacturingOrder {
       case ManufacturingOrder.status.Planned:
         return [
           {
-            title: I18n.t('Manufacturing_PlannedStartDate') + ':',
+            title: I18n.t('Manufacturing_PlannedStartDate') + ' :',
```

Comment:
This translation no longer exists.
Can you also check to possibility to merge MO and OP function or add an helper to simplify both functions please. It's pretty similar